fhirAssembly/src/test/java/io/vantiq/extsrc/fhirAssembly/TestFhirAssemblyOAuth.java

```diff
@@ -0,0 +1,112 @@
+package io.vantiq.extsrc.fhirAssembly;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import io.vantiq.client.Vantiq;
+import io.vantiq.client.VantiqResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * Test operations against a FHIR server using fhirConnection assembly
+ */
+@Slf4j
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class TestFhirAssemblyOAuth extends TestFhirAssembly {
+    
```

Comment:
Here, we use the same KeyCloak client that we use for the OAuth tests in ag2rs.  We know this is around in the KeyCloak instances where we run tests, so it makes it easy.